scripts/update_download_center.sh

```diff
@@ -35,7 +35,7 @@ echo "Setting up all Variables"
 UUID=${RANDOM}-${RANDOM}-${RANDOM}-${RANDOM}
 TMP_DIR=$(mktemp -d)
 CIRCLECI_API_BASE="https://circleci.com/api/v1.1/"
-NAVIT_DOWNLOAD_CENTER_REPO="git@github.com:navit-gps/download-center"
+NAVIT_DOWNLOAD_CENTER_REPO="git@github.com:OLFDB/download-center"
```

Comment:
please revert this

Reply:
Same as above. Would like to keep it in my fork only, so my CI build is working. Is that somehow possible?

Reply:
Need to revert d670c8ca19fa8976e389f6595420a29a810e9ed4 for this